src/4.3_to_5.0/legacySettingsFolder.ts

```diff
@@ -14,6 +14,58 @@ const user1Preferences = {
   deny: [],
   map: {
     "userFilters.enabled": true,
+    "widgets.Tabular.drillthrough.selectedColumns": {
+      "https://activerisk.zone1.scb.net:8081/server": {
+        MarketRiskCube: [
+          {
+            functionName: "MemberValue",
+            columnName: "riskCurrency",
+          },
+          {
+            functionName: "Caption",
+            columnName: "riskCurrency",
+          },
+          {
+            functionName: "MemberValue",
+            columnName: "subTradeId",
+          },
+          {
+            functionName: "Caption",
+            columnName: "subTradeId",
+          },
+        ],
+        SecondCube: [
+          {
+            functionName: "MemberValue",
+            columnName: "delta",
+          },
+          {
+            functionName: "Caption",
+            columnName: "delta",
+          },
```

Comment:
This is considered duplicates, so we are testing that part of the implementation ?

Reply:
This is considered duplicates indeed, but it's the first way to have duplicate I described above. Technically, I didn't test the second way of having duplicates.

I also tested the second way of having duplicates, as I added the column `subTradeId` in `MarketRiskCube` under two different servers.